src/ipred.rs

```diff
@@ -1985,6 +1984,7 @@ mod neon {
         };
     }
 
+    /// SAFETY: Must be called from [`angular_ipred::Fn::call`].
```

Comment:
```suggestion
    /// # Safety
    ///
    /// Must be called from [`angular_ipred::Fn::call`].
```

`# Safety` is used for docs, vs. `// SAFETY` for comments for `unsafe {}` blocks.